screen-orientation/lock-basic.html

```diff
@@ -1,64 +1,84 @@
 <!DOCTYPE html>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
 <script>
-test(() => {
-  screen.orientation.unlock();
-}, "Test that screen.orientation.unlock() doesn't throw when there is no lock");
+  test(() => {
```

Comment:
Just a nit: but please be sure to undo these whitespace changes. Otherwise, it will look like you changed everything :)